evals/registry.py

```diff
@@ -74,13 +76,15 @@ def is_chat_model(model_name: str) -> bool:
     if model_name in {"gpt-4-base"}:
         return False
 
-    CHAT_MODEL_NAMES = {"gpt-3.5-turbo", "gpt-4", "gpt-4-32k"}
+    CHAT_MODEL_NAMES = {"gpt-3.5-turbo", "gpt-3.5-turbo-16k", "gpt-4", "gpt-4-32k"}
+
     if model_name in CHAT_MODEL_NAMES:
         return True
 
-    for model_prefix in {"gpt-3.5-turbo-", "gpt-4-", "gpt-4-32k-"}:
+    for model_prefix in {"gpt-3.5-turbo-", "gpt-4-"}:
         if model_name.startswith(model_prefix):
             return True
+
```

Comment:
We may also want to throw an error when there's not an exact match, or be more restrictive about when the prefix matching applies to avoid similar errors in the future.